examples/amp-subscriptions-iframe.provider.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+<head>
+  <script src="../extensions/amp-access/0.1/iframe-api/build/index.js"></script>
+
+  <script>
+    
+    var pageConfig;
+    var pageProtocol;
+    function log() {
+      if (!console || !console.log) {
+        return;
+      }
+      var var_args = Array.prototype.slice.call(arguments, 0);
+      console.log.apply(console, var_args);
+    }
+
+    /** Implementation of AccessController */
+    var Controller = function() {
+    };
+    Controller.prototype.connect = function(origin, protocol, config) {
+      pageConfg = config; 
+      pageProtocol = protocol;
+      log('connect: ', origin, protocol, config);
+    };
+    Controller.prototype.authorize = function() {
+      log('authorize');
+      // This is the JSON expected in the article-access-iframe.amp.html.
+      if (pageProtocol === "amp-access") {
```

Comment:
Since this is specifically a `amp-subscriptions-...` sample, I think we should either remove `pageProtocol == 'amp-access'` or put it as a last `if`. Two goals here: (1) focus on subscriptions; (2) avoid anyone thinking that both are required. This could, of course, be solved by comments. So up to you.

Reply:
re-ordered, added clarifying comment